<launch>
    <include file="$(find samana)/launch/arduino_start.launch"/>
    <!--<include file="$(find samana)/launch/main_smach.launch"/>-->

    <!--<node name="pos_vel_plot"
        pkg="rqt_plot"
        type="rqt_plot"
        args="
                /arm_data/current_grabber
                /arm_data/current_lifter
             "
      />-->
    <!--<node name="rc_main" pkg="samana" type="rc_main.py" output="screen" respawn="true"/>-->
</launch>